### PR TITLE
Popup window problem in automation test.

### PR DESCRIPTION
**Pull request by PeterKuzyk**
Popup window problem in automation test. Asking for donation.

### CODE REVIEW

File: src/test/java/GroupBreakingBad.java [hunk 1]
```diff
@@ -17,11 +21,11 @@ public void nataliaVats() throws InterruptedException {
         Thread.sleep(3000);
 
         WebElement input = driver.findElement(By.xpath("//input[contains(@class, 'ph-a11y-location-box')]"));
-        Assert.assertEquals(input.getAttribute("placeholder"), "Enter City, State or Zip");
+        assertEquals(input.getAttribute("placeholder"), "Enter City, State or Zip");
```
Comment (sergeonevdah) [inline src/test/java/GroupBreakingBad.java:24]: You missed **Assert.** in line 24

File: src/test/java/GroupBreakingBad.java [hunk 2]
```diff
@@ -32,7 +36,25 @@ public void alexeySemenov() throws  InterruptedException{
         Thread.sleep(3000);
 
         WebElement menuElement = browser.findElement(By.xpath("//*[@id='CityList']/div[1]/div[2]/h4"));
-        Assert.assertEquals(menuElement.getText().toLowerCase(), "russia");
+        assertEquals(menuElement.getText().toLowerCase(), "russia");
```
Comment (sergeonevdah) [inline src/test/java/GroupBreakingBad.java:39]: You missed **Assert.** in line 39

**Review by sergeonevdah — COMMENTED** [review]
Check please )

**Review by PeterKuzyk — COMMENTED** [review]
done

File: src/test/java/GroupBreakingBad.java (hunk 1 above)
Comment (sergeonevdah) [reply src/test/java/GroupBreakingBad.java:24]: Its OK, your add line 10

File: src/test/java/GroupBreakingBad.java (hunk 2 above)
Comment (sergeonevdah) [reply src/test/java/GroupBreakingBad.java:39]: Its OK, your add line 10

**Review by sergeonevdah — APPROVED** [review]

**Comment by sergeonevdah** [comment]
close

**Pull request closed without merging**